docs/weaver_code_anatomy.md

```diff
@@ -5,53 +5,50 @@ sidebar_label: View the generated code
 slug: /generated_yaml
 ---
 
-🚧 This page is very much under construction as of 02/17/2021 as we develop the weaver's capabilities. 🚧
+:::warning
+This page is very much under construction as we develop the weaver's capabilities. These 🚧 emoji notes features that are being actively developed in the Weaver and thus might change
+:::
 
-The code [ALWeaver](https://apps-test.suffolklitlab.org/start/assemblylinewizard/assembly_line/#/1&new_session=1) generates is just a starting point. It uses the [labels and variables you added to your documents](doc_vars_reference.md) to make an interview that uses the AssemblyLine library.
+The code generated by [ALWeaver](https://apps-test.suffolklitlab.org/start/assemblylinewizard/assembly_line/#/1&new_session=1) is just a starting point. It uses the [labels and variables you added to your documents](doc_vars_reference.md) to make an interview that uses questions and features from the AssemblyLine library.
```

Comment:
No need to change this tiny thing, but just a note that `generated by` is passive voice.